esmvaltool/recipe_schema.yml

```diff
@@ -28,6 +28,8 @@ variable:
   reference_dataset: str(required=False)
   alternative_dataset: str(required=False)
   mip: str(required=False)
+  fx_mask: bool(required=False)
```

Comment:
This name is a bit confusing, since these variable can be used also for other purposes than masking. Actually, why do we need this switch? If `fx_variable` is provided, then it is automatically True, otherwise it is False.